elide-core/src/main/java/com/yahoo/elide/core/DataStoreTransaction.java

```diff
@@ -227,8 +227,8 @@ default void preCommit() {
         Object val = PersistentResource.getValue(entity, relationName, dictionary);
         if (val instanceof Collection) {
             Collection filteredVal = (Collection) val;
-            Optional<Sorting> sortingRules = Optional.ofNullable(sorting);
-            Optional<Pagination> paginationRules = Optional.ofNullable(pagination);
+            Optional<Sorting> sortingRules = Optional.ofNullable(sorting.isDefaultInstance() ? null : sorting);
```

Comment:
We should talk to @aklish and @duseja2. I think the default instance just isn't behaving as intended.